core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/util/deprecationMessages.kt

```diff
@@ -125,3 +125,15 @@ internal const val ALL_COLS_EXCEPT_REPLACE_VARARG = "this.allColsExcept { others
 internal const val EXCEPT_REPLACE = "this.except { other }"
 internal const val EXCEPT_REPLACE_VARARG = "this.except { others.toColumnSet() }"
 // endregion
+
+internal const val MOVE_TO_LEFT = "This `moveToLeft` overload will be removed in favor of `moveToStart`. $MESSAGE_0_16"
```

Comment:
please move them to the right region so they're easier to keep track of

Reply:
done